src/shim/umq/hwctx.cpp

```diff
@@ -61,16 +61,16 @@ fini_log_buf(void)
 
 void
 hw_ctx_umq::
-set_metadata(int num_cols, size_t size, uint64_t bo_paddr, enum umq_log_flag flag)
```

Comment:
this is for the LOG_BUFFER setting.
we just add the define for UMQ_LOG_BUFFER in this PR, the what is the flag passed in to this call?

I guess we are still using UMQ_DEBUG_BUFFER?

Reply:
that should be UMQ_LOG_BUFFER